core/js/typespecs/app_types.ts

```diff
@@ -146,29 +146,36 @@ export const datasetSchema = z
 export type Dataset = z.infer<typeof datasetSchema>;
 
 const promptBaseSchema = generateBaseTableSchema("prompt");
-export const promptSchema = z.strictObject({
-  id: promptBaseSchema.shape.id,
-  // This has to be copy/pasted because zod blows up when there are circular dependencies
-  _xact_id: z
-    .string()
-    .describe(
-      `The transaction id of an event is unique to the network operation that processed the event insertion. Transaction ids are monotonically increasing over time and can be used to retrieve a versioned snapshot of the prompt (see the \`version\` parameter)`,
-    ),
-  project_id: promptBaseSchema.shape.project_id,
-  log_id: z
-    .literal("p")
-    .describe("A literal 'p' which identifies the object as a project prompt"),
-  org_id: organizationSchema.shape.id,
-  name: promptBaseSchema.shape.name,
-  slug: z.string().describe("Unique identifier for the prompt"),
-  description: promptBaseSchema.shape.description,
-  created: promptBaseSchema.shape.created,
-  prompt_data: promptDataSchema
-    .nullish()
-    .describe("The prompt, model, and its parameters"),
-  tags: z.array(z.string()).nullish().describe("A list of tags for the prompt"),
-  metadata: promptBaseSchema.shape.metadata,
-});
+export const promptSchema = z
+  .strictObject({
+    id: promptBaseSchema.shape.id,
+    // This has to be copy/pasted because zod blows up when there are circular dependencies
+    _xact_id: z
+      .string()
+      .describe(
+        `The transaction id of an event is unique to the network operation that processed the event insertion. Transaction ids are monotonically increasing over time and can be used to retrieve a versioned snapshot of the prompt (see the \`version\` parameter)`,
+      ),
+    project_id: promptBaseSchema.shape.project_id,
+    log_id: z
+      .literal("p")
+      .describe(
+        "A literal 'p' which identifies the object as a project prompt",
+      ),
+    org_id: organizationSchema.shape.id,
+    name: promptBaseSchema.shape.name,
+    slug: z.string().describe("Unique identifier for the prompt"),
+    description: promptBaseSchema.shape.description,
+    created: promptBaseSchema.shape.created,
+    prompt_data: promptDataSchema
+      .nullish()
+      .describe("The prompt, model, and its parameters"),
+    tags: z
+      .array(z.string())
+      .nullish()
+      .describe("A list of tags for the prompt"),
+    metadata: promptBaseSchema.shape.metadata,
+  })
+  .openapi("Prompt");
```

Comment:
Just adding a name to this type, so stainless can generate a `Prompt` object in its SDKs. This also cuts down on some duplication in the openAPI spec.